src/model/gui/cli/material/person/Person.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace model\gui\cli\material\person;
+
+use model\gui\cli\material\AbstractMaterial;
+use interfaces\model\data\position\PositionInterface;
+use interfaces\repository\output\PrintRepositoryInterface;
+use interfaces\model\method\material\person\PersonInterface;
+use interfaces\model\method\action\speak\SpeakInterface;
+
+/**
+ *
+ * @author kevinfrantz
+ *        
+ */
+class Person extends AbstractMaterial implements PersonInterface, SpeakInterface
```

Comment:
Eine Person ist doch kein Material 🐙

Reply:
Personen bestehen aus Materie. Ob diese Kategorisierung in dieser Form sinnvoll ist, ist natürlich eine andere Frage. Vielleicht besteht eine Person aus einer Dependency Injection Materie, sowie Seele ;)